spec/integration/integration_spec.rb

```diff
@@ -901,5 +901,28 @@ class Tag
         end
       end
     end
+
+    describe 'overriden changes_method with non_database_fields' do
```

Comment:
There should be a test with a tracked field is a method, and a `attr_accessor` or similar. There should be a test for all fields + a custom field, all fields being the default, and just a custom field.